this is inside a windows block so commenting it is fine
--
--- a/configure.ac
+++ b/configure.ac
@@ -6898,7 +6898,7 @@
         WINDOWS_SDK_LIB_SUBDIR=$winsdklibsubdir
         # The variable also contains the Windows SDK version
         echo $WINDOWS_SDK_LIB_SUBDIR
-        IFS='.' read -r SDK_v1 SDK_v2 SDK_v3 SDK_v4 <<< "$WINDOWS_SDK_LIB_SUBDIR"
+        #IFS='.' read -r SDK_v1 SDK_v2 SDK_v3 SDK_v4 <<< "$WINDOWS_SDK_LIB_SUBDIR"
         # Assuming maximum of 5 digits for each part and ignoring last part
         SDK_NORMALIZED_VER=$((SDK_v1 * 10000000000 + SDK_v2 * 100000 + SDK_v3))
         # 10.0.20348.0 is the minimum required version
